xunit-autolabeler-v2/ast_parser/python/source_parsers/direct_invocation.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Any, List
+
+
+from ast_parser.python.drift_data_dict import make_drift_data_dict
```

Comment:
Use import  statements for packages and modules only, not for individual classes or functions (typing is an exemption so don't worry about that one).

Reply:
For `constants`, which of the following is better?
1) Leave as-is
2)`import ... as lib_constants`

(The reason I ask: there are other constants files, and I want to make sure we disambiguate them.)

Reply:
Resolved offline. 🙂